src/strategies/dcb-call-tiers/index.ts

```diff
@@ -0,0 +1,45 @@
+import { formatUnits } from '@ethersproject/units';
+import { multicall } from '../../utils';
+
+export const author = 'jaymaree';
+export const version = '0.1.0';
+
+function getArgs(options, address: string) {
+  const args: Array<string | number> = options.args || ['%{address}'];
+  return args.map((arg) =>
+    typeof arg === 'string' ? arg.replace(/%{address}/g, address) : arg
+  );
+}
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+) {
+  const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
+  const response = await multicall(
+    network,
+    provider,
+    [options.methodABI],
+    addresses.map((address: any) => [
+      options.address,
+      options.methodABI.name,
+      getArgs(options, address)
+    ]),
+    { blockTag }
+  );
+  return Object.fromEntries(
+    response.map((value, i) => [
+      addresses[i],
+      parseFloat(
+        formatUnits(
+          options?.output ? value[options.output].toString() : value.toString(),
+          options.decimals
+        )
+      )
+    ])
+  );
```

Comment:
Hi @mj-dcb I think you can use contract-call strategy instead, can try? It is not required to create a new strategy if it works :)

Reply:
@ChaituVR, you're right, we could use contract-call right now, but we will change the strategy in the short future therefore, we wanted to add our base strategy with a working example.json file.